src/components/favorite-button/favorite-button.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-tooltip left>
+    <template v-slot:activator="{ on }">
+      <div @click="handleSetFavorite" v-on="on">
+        <v-icon v-if="favorite" color="yellow">mdi-star</v-icon>
+        <v-icon v-else color="grey lighten-1">mdi-star-outline</v-icon>
+      </div>
+    </template>
+    <span>{{tooltipText}}</span>
+  </v-tooltip>
+</template>
```

Comment:
Please recall we're not using single file components, must split the code into three different files

Reply:
It's too short to split it into multiple files, isn't it? But I can do if necessary.

Reply:
This is a defined standard, we agreed this directive  on the first day